lib/cppcheck.cpp

```diff
@@ -1954,6 +1962,11 @@ void CppCheck::analyseClangTidy(const FileSettings &fileSettings)
     }
 #endif
 
+    if (!mExecuteCommand) {
+        std::cerr << "Failed to execute '" << exe << "' (no command callback provided)" << std::endl;
+        return;
+    }
+
```

Comment:
Reporting a proper error is already tracked in https://trac.cppcheck.net/ticket/13827.